bitrise.yml

```diff
@@ -248,7 +248,7 @@ workflows:
     meta:
       bitrise.io:
         stack: osx-xcode-14.1.x-ventura
-        machine_type_id: g2-m1.8core
+        machine_type_id: g2.4core
```

Comment:
Looks like there is an issue with M1 ventura.. Using a regular Intel works better and we are not seeing the Lost connection error

Reply:
> Looks like there is an issue with M1 ventura.. Using a regular Intel works better and we are not seeing the Lost connection error

😨 Have we seen more `Lost connection error` since we use the `osx-xcode-14.1.x-ventura` stack?

Reply:
not in those because they are intel too, the issue seen is with M1 ventura